--- a/Makefile
+++ b/Makefile
@@ -516,6 +516,7 @@
 		-I$(objtree)/arch/$(SRCARCH)/include/generated \
 		$(if $(building_out_of_srctree),-I$(srctree)/include) \
 		-I$(objtree)/include \
+		-I$(objtree)/bsp/include \
 		$(USERINCLUDE)
 
 KBUILD_AFLAGS   := -D__ASSEMBLY__ -fno-PIE
@@ -676,6 +677,7 @@
 drivers-$(CONFIG_SAMPLES) += samples/
 drivers-$(CONFIG_NET) += net/
 drivers-y	+= virt/
+drivers-y	+= bsp/drivers/
 libs-y		:= lib/
 endif # KBUILD_EXTMOD
 
